### PR TITLE
ENH: Add examples to the template.

### LINKED ISSUE #34: Add `examples` folder content to the module template
See:

  https://github.com/KitwareMedical/ITKMorphologicalContourInterpolation/tree/master/examples

for what the content can be like.

### PR DESCRIPTION
**Pull request by jhlegarreta**
Add a hint to foster writing examples for the contributed classes.

Resolves #34.

### CODE REVIEW

**Comment by jhlegarreta** [comment]
Superseds PR #46 (could not reopen it as it was branched from to merge into `master`).

A CI job should also run the example I guess. I do not have the bandwidth to add that.

File: {{cookiecutter.project_name}}/examples/{{ cookiecutter.example_name }}.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+#        The assumption is that...
+#  parameter_2: controls the..
+#        A tradeoff between...
+
+import sys
+import itk
+
+import numpy as np
+
+
```
Comment (thewtex) [inline {{cookiecutter.project_name}}/examples/{{ cookiecutter.example_name }}.py:34]: May want to use `argparse` here.

**Review by thewtex — COMMENTED** [review]
@jhlegarreta awesome!

CI update being tested in #162

**Comment by thewtex** [comment]
> CI update being tested in #162

Rebase should address CI

File: {{cookiecutter.project_name}}/examples/{{ cookiecutter.example_name }}.py (hunk 1 above)
Comment (jhlegarreta) [reply {{cookiecutter.project_name}}/examples/{{ cookiecutter.example_name }}.py:33]: Done in fae6465.

**Review by thewtex — APPROVED** [review]
:book:

**Review by dzenanz — APPROVED** [review]

**Pull request merged**